util/admin/admin_test.go

```diff
@@ -466,6 +418,8 @@ func (s *testSuite) testIndex(c *C, tb table.Table, idx table.Index) {
 	// table     data (handle, data): (1, 10), (2, 20), (3, 30)
 	key = tablecodec.EncodeRowKey(tb.Meta().ID, codec.EncodeInt(nil, 4))
 	txn.Delete(key)
+	key = tablecodec.EncodeRowKey(tb.Meta().ID, codec.EncodeInt(nil, 5))
```

Comment:
Keep it consistent with the comments.
And I want table count is different with index count.